packages/flutter_tools/lib/src/test/event_printer.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:convert' show JSON;
```

Comment:
"Chromium Authors" is a legal term; I think we have some license deps to free ourselves of first before we can become the "Flutter Authors"... @sethladd would know the full story.